ultraScurve.py

```diff
@@ -140,51 +98,52 @@
             trimVal = (0x3f & readVFAT(ohboard,options.gtx,vfat,"VFATChannels.ChanReg%d"%(scCH)))
             writeVFAT(ohboard,options.gtx,vfat,"VFATChannels.ChanReg%d"%(scCH),trimVal)
 
+    gemData.mode[0] = scanmode.SCURVE
```

Comment:
Consider having the `gemData` constructor to take the `scanmode.<TYPE>` as a parameter

Reply:
Done